scripts/cdc_prams/process.py

```diff
@@ -0,0 +1,758 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The Python script loads the datasets,
+cleans them and generates the cleaned CSV, MCF and TMCF file.
+"""
+import os
+import sys
+import re
+from copy import deepcopy
+import pandas as pd
+import numpy as np
+from absl import app, flags
+# from util.alpha2_to_dcid import COUNTRY_MAP
+import tabula as tb
+
+_CODEDIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(1, _CODEDIR)
+# pylint: disable=import-error
+# pylint: disable=wrong-import-position
+sys.path.insert(1, os.path.join(_CODEDIR, '../../util/'))
+from statvar_dcid_generator import get_statvar_dcid
+from state_division_to_dcid import _PLACE_MAP
+
+sys.path.insert(1, os.path.dirname(os.path.abspath(__file__)))
+from statvar import statvar_col
+from constants import (_MCF_TEMPLATE, _TMCF_TEMPLATE, DEFAULT_SV_PROP, _PROP,
+                       _TIME, _INSURANCE, _PROP4, _PV_PROP, _YEAR)
+# pylint: enable=import-error
+# pylint: enable=wrong-import-position
+_FLAGS = flags.FLAGS
+default_input_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                  "input_files")
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+
+def _multiline(df, flag):
```

Comment:
Please provide different name for function.
Replace flag with geo in all instances. Make geo as enum with values National and State.
Incase flag is not used in function, then remove it.

Reply:
Replaced flag with geo.